.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 
 # These owners will be the default owners for everything in the repo.
 #*       @defunkt
-*       @MatthewPyle-NOAA @ShunLiu-NOAA @JacobCarley-NOAA @chan-hoo @BenjaminBlake-NOAA @RatkoVasic-NOAA @JiliDong-NOAA @hu5970
+*       @NOAA-EMC/RRFS/RRFS-devs
```

Comment:
I know some work was done to update the CODEOWNERS file, but GitHub is still saying the file contains errors.  Should we revert back to the previous version which listed our individual usernames?

Reply:
Done